python/paddle/incubate/optimizer/modelaverage.py

```diff
@@ -11,12 +11,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
 
 import paddle
 from paddle import _C_ops
 from paddle.base import framework
 from paddle.base.dygraph import base as imperative_base
-from paddle.base.layer_helper import LayerHelper
+from paddle.base.layer_helper import LayerHelper, LayerHelperBase
```

Comment:
```suggestion
from paddle.base.layer_helper import LayerHelper
```

貌似不需要 `LayerHelperBase` 吧